bot/extensions/vote_cog.py

```diff
@@ -0,0 +1,315 @@
+from typing import Callable
+from discord.ext.commands import Cog, Context, hybrid_command, hybrid_group
+from bot.extensions.command_error_handler import send_command_help
+from collections.abc import Sequence
+from discord import Embed
+from discord.ui import Button, View
+import discord
+import random
```

Comment:
except if I missed it, but this library is never used in your code

Reply:
You're right :)